Menu/GameShell/10_Settings/Time/pages.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*- 
+
+from timezone_lib_list_page import TimezoneListPage
+import myvars
+
+def InitTimePage(main_screen):
+    myvars.TimePage = None
```

Comment:
I would like to add a generic clock page with the options for 12 or 24 hour display.
I haven't yet looked into the code for the top-bar, though.